### PR TITLE
Add POLYGON and new transfer time format supportion

### PR DESCRIPTION
**Pull request by Kozer4**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -423,12 +424,15 @@ export class AllbridgeCoreSdk {
     destinationChainToken: TokenInfoWithChainDetails,
     messenger: Messenger
   ): number | null {
-    const sourceTxTime = sourceChainToken.txTime[messenger]?.in;
-    const destinationTxTime = destinationChainToken.txTime[messenger]?.out;
-    if (!sourceTxTime || !destinationTxTime) {
+    const messengerTransferTime: MessengerTransferTime | undefined =
```
Comment (Kozer4) [inline src/index.ts:427]: pay close attention to changes to this method

**Review by m-cote — APPROVED** [review]
Looks good to me.

File: src/index.ts [hunk 2]
```diff
@@ -423,12 +424,15 @@ export class AllbridgeCoreSdk {
...
+      sourceChainToken.transferTime[
```
Comment (YaroslavNekryach) [inline src/index.ts:428]:
```suggestion
    return sourceChainToken.transferTime?.[destinationChainToken.chainSymbol as ChainSymbol]?.[messenger] ?? null;
```

Comment (Kozer4) [reply src/index.ts]: thx, imlemented

**Pull request merged**